helm/charts/kratos/templates/_helpers.tpl

```diff
@@ -41,9 +41,9 @@ Generate the dsn value
 {{/*
 Generate the secrets.session value
 */}}
-{{- define "kratos.secrets.session" -}}
-{{- if .Values.kratos.config.secrets.session -}}
-{{- .Values.kratos.config.secrets.session }}
+{{- define "kratos.secrets.cookie" -}}
```

Comment:
You might also want to change default `secrets.session` in `values.yaml`.

Reply:
Or just remove it altogether. That way, user is forced to explicitly set the config value instead of relying on unsafe defaults.

Reply:
`secrets.default` is also missing :)